src/applications/facility-locator/api/LocatorApi.js

```diff
@@ -1,5 +1,6 @@
 import compact from 'lodash/compact';
 import { api } from '../config';
+import { fetch } from '../../../platform/utilities/api';
```

Comment:
Not required, but you can actually omit the path to the directory and just lead with `platform/`. It makes imports across multiple files a little easier.